csatar-plugins/csatar/models/MandateType.php

```diff
@@ -108,12 +97,15 @@ function beforeDelete()
     {
```

Comment:
Ez működik soft-delete-el is?

Reply:
Igen, soft delete van a mandate_type tablaban.